src/app/base/components/ArrowPagination/ArrowPagination.test.tsx

```diff
@@ -13,10 +13,13 @@ describe("ArrowPagination", () => {
       />
     );
 
-    expect(screen.getByRole("button", { name: Labels.GoBack })).toBeDisabled();
+    expect(screen.getByRole("button", { name: Labels.GoBack })).toHaveAttribute(
+      "aria-disabled",
+      "true"
+    );
```

Comment:
```suggestion
    expect(screen.getByRole("button", { name: Labels.GoBack })).toBeAriaDisabled();
```

Reply:
Huh, I don't know how my find + replace didn't catch that. Will update.